spec/jobs/http_fetcher_job_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe HttpFetcherJob do
+  include ActiveJob::TestHelper
+
+  let(:source) { create(:source, url: 'http://example.com/apis') }
```

Comment:
Definitely a nitpick but the url could perhaps be extracted into its own `let` since we use it again for the stub below - would be a little more robust 😄